tests/zfs-tests/tests/functional/procfs/cleanup.ksh

```diff
@@ -26,4 +26,8 @@
 
 . $STF_SUITE/include/libtest.shlib
 
+if ! is_linux ; then
```

Comment:
This if block (here and in the tests) shouldn't be required, because it already exists in the setup test. When the setup test fails, the other tests should be skipped.

Reply:
I've removed these checks. The procfs tests are in linux.run so they only run on Linux now.